src/audio/src/src_common.c

```diff
@@ -531,9 +531,9 @@ int src_params_general(struct processing_module *mod,
 	}
 
 	/* free any existing delay lines. TODO reuse if same size */
-	rfree(cd->delay_lines);
+	mod_free(mod, cd->delay_lines);
```

Comment:
...if automatic freeing is a feature, then we can drop all these `mod_free()` calls?